kolibri/plugins/coach/assets/src/views/recent-page/channel-list.vue

```diff
@@ -28,10 +32,44 @@
 
   module.exports = {
     name: 'channelList',
-    $trNameSpace: 'channelList',
+    $trNameSpace: 'coachRecentPageChannelList',
     $trs: {
       channels: 'Channels',
       channelList: 'Channel list',
+      lastActive: 'Last active',
+      daysPassed: 'X day(s) ago',
+      monthsPassed: 'X month(s) ago',
+      timePassed:
+      '{amount, number} {measure, select, ' +
+        'day {' +
+          '{amount, plural,' +
+            'one {day}' +
+            'other {days}' +
+          '}' +
+        '}' +
+        'month {' +
+          '{amount, plural,' +
+            'one {month}' +
+            'other {months}' +
+          '}' +
+        '}' +
+      '} ago',
```

Comment:
turns out it's not necessary: https://github.com/learningequality/kolibri/pull/1184